FirebaseVertexAI/Sources/Safety.swift

```diff
@@ -90,18 +90,41 @@ public struct SafetyRating: Equatable, Hashable, Sendable {
 @available(iOS 15.0, macOS 11.0, macCatalyst 15.0, tvOS 15.0, watchOS 8.0, *)
 public struct SafetySetting {
   /// Block at and beyond a specified ``SafetyRating/HarmProbability``.
-  public enum HarmBlockThreshold: String, Sendable {
-    // Content with `.negligible` will be allowed.
-    case blockLowAndAbove = "BLOCK_LOW_AND_ABOVE"
+  public struct HarmBlockThreshold: EncodableProtoEnum, Sendable {
```

Comment:
Does the struct need to conform to `Comparable`?

Reply:
It would be feasible to make it conform to `Comparable` since it does have a natural order but I think we should wait for that feature to be requested.

Note: This value is set by the developer, not received in a response, so they don't need to compare if a received value is above or below a certain threshold.